types/react-native/index.d.ts

```diff
@@ -8151,6 +8152,42 @@ export interface VibrationStatic {
     cancel(): void;
 }
 
+export namespace Appearance {
+    type ColorSchemeName = 'light' | 'dark';
+
+    type AppearancePreferences = {
+        colorScheme: ColorSchemeName | null | undefined;
+    };
+
+    type AppearanceListener = (preferences: AppearancePreferences) => void;
+
+    /**
+     * Note: Although color scheme is available immediately, it may change at any
+     * time. Any rendering logic or styles that depend on this should try to call
+     * this function on every render, rather than caching the value (for example,
+     * using inline styles rather than setting a value in a `StyleSheet`).
+     *
+     * Example: `const colorScheme = Appearance.getColorScheme();`
+     *
+     * @returns {ColorSchemeName | null | undefined} Value for the color scheme preference.
+     */
+    export function getColorScheme(): ColorSchemeName | null | undefined;
+
+    /**
+     * Add an event handler that is fired when appearance preferences change.
+     *
+     * @returns {AppearancePreferences} Preferences object containting value for the color scheme.
```

Comment:
These jsdoc annotations, the type part at least, aren’t really necessary. Interestingly, though, the type here doesn’t line up with the actual returned type, what gives?

Reply:
Fixed the test by removing the returns lines, and yeah I just messed up in the type at first. 😅